package.json

```diff
@@ -42,6 +42,7 @@
     "react-fast-marquee": "^1.6.0",
     "react-query": "^3.39.3",
     "react-tabs": "^6.0.2",
+    "react-toastify": "^10.0.4",
```

Comment:
let's get rid of 3rd party plugin. we should be able to do a component from scratch.